tests/integration_tests/performance/test_block_performance.py

```diff
@@ -25,6 +25,7 @@
 import framework.stats as st
 from integration_tests.performance.configs import defs
 from integration_tests.performance.utils import handle_failure
+from integration_tests.security.test_spectre import test_spectre_mitigations
```

Comment:
If you move it to conftest.py, then it does not need to get explicitly imported.